osu.Game.Rulesets.Mania/Difficulty/ManiaDifficultyAttributes.cs

```diff
@@ -19,6 +19,27 @@ public class ManiaDifficultyAttributes : DifficultyAttributes
         [JsonProperty("great_hit_window")]
         public double GreatHitWindow { get; set; }
 
+        /// <summary>
+        /// The perceived overall difficulty of the map.
+        /// </summary>
+        [JsonProperty("overall_difficulty")]
+        public double OverallDifficulty { get; set; }
+
+        /// <summary>
+        /// The number of notes in the beatmap.
+        /// </summary>
+        public int NoteCount { get; set; }
+
+        /// <summary>
+        /// The number of hold notes in the beatmap.
+        /// </summary>
+        public int HoldNoteCount { get; set; }
+
+        /// <summary>
+        /// Conversion status from standard.
+        /// </summary>
+        public bool Convert { get; set; }
```

Comment:
nit: change to `IsConvert`, should yield better readibility when doing `if (attributes.IsConvert)`

Reply:
Done.